pylib/gyp/easy_xml.py

```diff
@@ -121,7 +122,10 @@ def WriteXmlIfChanged(content, path, encoding="utf-8", pretty=False, win32=False
 
     default_encoding = locale.getdefaultlocale()[1]
     if default_encoding and default_encoding.upper() != encoding.upper():
-        xml_string = xml_string.encode(encoding)
+        if sys.platform == "win32" and sys.version_info < (3, 7):
```

Comment:
Can we use the existing `win32` variable?

Reply:
Thank you for feedback) I have an idea on this topic but I need to make prove of concept. Cause I am noob in python, can u tell me pls, how can i test programm locally via unit tests and by hand?) thanks in advance